libs/blocks/global-navigation/global-navigation.js

```diff
@@ -526,26 +527,32 @@ class Gnav {
 
     const onAnalyticsEvent = (data) => {
       if (!data) return;
-
       const getInteraction = () => {
-        const {
-          event: { type, subtype } = {},
-          source: { name } = {},
-          content: { name: contentName } = {},
-        } = data;
-
-        switch (`${name}|${type}|${subtype}|${contentName || ''}`) {
-          case 'profile|click|sign-in|':
+        const { workflow, type, subtype } = data.event || data;
```

Comment:
the previous `source.name` property was lower-case, while the `workflow` property has the first letter upper-case. Using `workflow` without any transformation will break the existing use-cases for profile / appswitcher.

Reply:
the previous `source.name` property returned `appswitcher`, while `workflow` returns `app switcher`. Using workflow without trimming will break the existing use-cases for appswitcher.

Reply:
Thanks for pointing out 👍 I made the changes to accommodate this